crates/cli/src/common_args.rs

```diff
@@ -0,0 +1,15 @@
+use clap::Arg;
```

Comment:
nit @jdetter does this belong more in `src/` or `src/subcommands`?

Reply:
This is fine, leave it as is 👍